test/distributed/org/apache/cassandra/distributed/test/FrozenUDTTest.java

```diff
@@ -124,6 +125,7 @@ public void testUpgradeSStables() throws IOException
         }
     }
 
+    /* See CASSANDRA-19764 */
```

Comment:
The test was consistency failing with the trie implementation, because we can't use a temporary broken ordering to insert rows (since the row key in the trie is a function of the type). With other implementations we could insert one row (due to [this change](https://github.com/apache/cassandra/commit/2f0eb6f799f32c6f01d1f8384d48910c34ff6a98#diff-2f66334dab1923767bcb02c37ae8009cab617d21c4586c5a34be8803aac0263bR103)) before getting corruption.

Reply:
Ok, so the original test was also failing, and the change here is just to demonstrate the failure with the old memtable type?

Overall, the change as is makes it sound like having the test error out is the end goal, but I'd argue it is a bug to be fixed by CASSANDRA-19764, just not a new one. If that understanding is correct, I'd have rather marked the test as skipped with a quick explanation to clarify there is something not quite ok here.